backend/backend/check_position.py

```diff
@@ -0,0 +1,60 @@
+""" Module providing functionality to check the type of space the player is
+    on. """
+
+import sys
+import json
+import backend.player
+import backend.game
+import backend.properties
+import backend.miscellaneous
+from backend.charge_rent import charge_rent
+from backend.pay_tax import pay_tax
+
+
+def check_position(source=sys.stdin):
+    """Check the type of space the player is on.
+
+    The two types of space this function recognises are properties and
+    miscellaneous (chance, tax, etc.).
+
+    """
+    # Get the player id from the client
+    details = json.load(source)
+    player_id = details["player_id"]
```

Comment:
Since we’re no longer expecting this function to be called by the client, I think this should take the player id as an argument rather than loading it from stdin – does that make sense?